website/docs/for-android/getting-started.md

```diff
@@ -10,23 +10,21 @@ Once you [obtain an API key](./guide#signup) and [install Ionic Portals](./guide
 
 ## Creating a Custom Application Class
 
-In Android, you will have to register your Portals instance and start creating Portals via the [PortalManager](../reference/android/portal-manager). To do this, a custom [Application](https://developer.android.com/reference/android/app/Application) class is recommended. In this Application class, you can override `Application#onCreate()` to register and create Portals.
-
-<Tabs 
-    defaultValue="kt" 
-    values={[
-        { label: 'Kotlin', value: 'kt', },
-        { label: 'Java', value: 'java', },
-    ]}
->
-<TabItem value="kt">
+In Android, you will have to register your Portals instance and start creating Portals via the [PortalManager](./reference/api/portal-manager). To do this, a custom [Application](https://developer.android.com/reference/api/app/Application) class is recommended. In this Application class, you can override `Application#onCreate()` to register and create Portals.
```

Comment:
anywhere that `android` was swapped with `api` in the `developer.android.com` links broke them in this file - theres a few just tagging the first one I see